template/Dockerfile

```diff
@@ -0,0 +1,146 @@
+# incompatible with Drupal < 8.
+# @todo: Is the launcher even needed? We add /var/www/vendor/bin on the global path.
+ARG DRUSH_LAUNCHER_VERSION=0.6.0
+ARG DRUSH_LAUNCHER_SHA256=c3f32a800a2f18470b0010cd71c49e49ef5c087f8131eecfe9b686dc1f3f3d4e
+RUN curl -fsSOL https://github.com/drush-ops/drush-launcher/releases/download/$DRUSH_LAUNCHER_VERSION/drush.phar \
+  && echo "$DRUSH_LAUNCHER_SHA256 drush.phar" | sha256sum -c - \
+  && mv drush.phar /usr/bin/drush && chmod +x /usr/bin/drush
+
+# Pre-trust Github host certificates.
+RUN ssh-keyscan github.com >> /etc/ssh/ssh_known_hosts
+
+ADD templates /templates
+ADD scripts/* /usr/local/bin/
+
+# /common-php70
+{{ if .Env.IS_DEV }}
+# NodeJS + Yarn
+ARG NODE_MAJOR_VERSION=10
+RUN apt-get update \
+  && apt-get install -y apt-transport-https lsb-release gnupg > /dev/null 2>&1 \
+  && curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - \
+  && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+  && echo "deb https://deb.nodesource.com/node_$NODE_MAJOR_VERSION.x $(lsb_release --codename | cut -f2) main" > /etc/apt/sources.list.d/nodesource.list \
+  && echo "deb-src https://deb.nodesource.com/node_$NODE_MAJOR_VERSION.x $(lsb_release --codename | cut -f2) main" >> /etc/apt/sources.list.d/nodesource.list \
+  && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
+  && apt-get update \
+  && apt-get install -y nodejs \
+  && apt-get install -y --no-install-recommends yarn \
+  && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# XDebug
+RUN pecl install xdebug \
+  && docker-php-ext-enable xdebug \
+  && echo "xdebug.default_enable=0\nxdebug.coverage_enable=0" > $PHP_INI_DIR/conf.d/xdebug.ini
+
+# Blackfire Probe. Note: We do not install Blackfire CLI. You will
+# need that in order to trigger Blackfire tests. The probe just instruments
+# things on the server side.
+RUN version=$(php -r "echo PHP_MAJOR_VERSION.PHP_MINOR_VERSION;") \
+    && curl -A "Docker" -o /tmp/blackfire-probe.tar.gz -D - -L -s https://blackfire.io/api/v1/releases/probe/php/linux/amd64/$version \
+    && mkdir -p /tmp/blackfire \
+    && tar zxpf /tmp/blackfire-probe.tar.gz -C /tmp/blackfire \
+    && mv /tmp/blackfire/blackfire-*.so $(php -r "echo ini_get('extension_dir');")/blackfire.so \
+    && printf "extension=blackfire.so\nblackfire.agent_socket=tcp://blackfire:8707\n" > $PHP_INI_DIR/conf.d/blackfire.ini \
+    && rm -rf /tmp/blackfire /tmp/blackfire-probe.tar.gz
+
+# Terminus.
+ARG TERMINUS_VERSION=2.2.0
+ARG TERMINUS_SHA256="73fcdf6ceee23731c20bff45f668bde09230af347670a92e5ca97c2c008ae6e0"
+RUN curl -fsSOL https://github.com/pantheon-systems/terminus/releases/download/$TERMINUS_VERSION/terminus.phar \
+  && echo "$TERMINUS_SHA256 terminus.phar" | sha256sum -c - \
+  && mv terminus.phar /usr/local/bin/terminus \
+  && chmod +x /usr/local/bin/terminus
+
+# Gulp-cli.
+RUN npm i --no-cache -g gulp-cli
+ADD Dockerfile /tmp
```

Comment:
Wondering what the point of this is?

Reply:
Ha, oh - I was trying to bust a cache.  Will remove - thanks.